tests/run-jbang-tests.sh

```diff
@@ -0,0 +1,346 @@
+#!/bin/bash
+
+script_dir=$(dirname "$0")
+
+executionMode=$1
+
+test_failure=0
+test_count=0
+TEST_TIMEOUT=20
+
+test_dir=tests/target/jbang/
+
+glow_version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+# find the latest version of WildFLy that is supported by WildFly Glow
+wildfly_latest_version=$(curl -s 'https://raw.githubusercontent.com/wildfly/wildfly-galleon-feature-packs/refs/heads/release/versions.yaml'  | yq .latest)
+
+function setupTestSuite {
+  echo Clear JBang cache
+  jbangExec cache clear
+  mkdir -p $test_dir
+}
+
+function tearDownTestSuite {
+  rm -rf $test_dir
+}
+
+function check_endpoint() {
+    local url=$1
+
+    response=$(curl -s -w "%{http_code}" $url)
```

Comment:
Probably not a huge deal, but it probably should be `"${url}"`, see https://www.shellcheck.net/wiki/SC2086.